core/controlplane/config/templates/stack-template.json

```diff
@@ -461,6 +488,11 @@
             "Key": "Name",
             "PropagateAtLaunch": "true",
             "Value": "{{$.ClusterName}}-{{$.StackName}}-kube-aws-etcd-{{$etcdIndex}}"
+          },
+          {
+            "Key": "kube-aws:role",
```

Comment:
reason I asked is because there can be 10 Tags on a resource and company policies might require 4-5 of them to be set, so no much room for new tags :) I'd leave it simple then, until it becomes a problem